Set/Card/CardViewModel.swift

```diff
@@ -14,6 +14,10 @@ class CardViewModel: ObservableObject {
         self.isColorBlindModeEnabled = isColorBlindModeEnabled
     }
     
+    var cardIsFaceUp: Bool {
```

Comment:
Better to start names of `Bool` with `is`

Reply:
fixed